code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -50,6 +50,19 @@ In short:
 
 	suspend_alert = 1
 
+	OnEnter1()
+	OnEnter2()
+	OnEnter3()
+	OnEnter4()
+	OnEnter5()
```

Comment:
Proc names could be way more explanatory.
Otherwise, I can predict what'll happen to this file:
>"why the fuck are all these procs for no reason?"
>"I'm putting this in one proc"

Why are these individual procs?